frontend/web/components/modals/Rule.js

```diff
@@ -65,7 +65,8 @@ export default class Rule extends PureComponent {
                                   data-test={`${this.props['data-test']}-value-${i}`}
                                   className="input-container--flat full-width"
                                   value={`${value}`}
-                                  placeholder="Value *"
+                                  placeholder={ operator && operator === 'IS_SET'|| operator && operator === 'IS_NOT_SET' ? 'Value (N/A)' : 'Value *'}
```

Comment:
Nice work, although I think this could be slightly more succinct still, right? Either of the following options should work: 

```javascript
{operator && (operator === IS_SET || operator === IS_NOT_SET) ? 'Value (N/A)' : 'Value *'}
```

```javascript
{operator && [IS_SET, IS_NOT_SET].includes(operator) ? 'Value (N/A)' : 'Value *'}
```

Probably the first of these options is the more obvious but if we end up with a third, we may want to update it to use the second option (or @kyle-ssg will have developed a more general solution). 

Also, it's important to try and be consistent with your code style. For example, you've left a space after the `{` symbol for the `placeholder` definition, but not at the end of the ternary statement and also, there are no spaces in the logic for the `disabled` attribute. Note that the rest of the code doesn't leave any spaces before / after the `{` / `}` so let's follow that.